dataproc/quickstart/quickstart.py

```diff
@@ -125,4 +124,23 @@ def quickstart(project_id, region, cluster_name, job_file_path):
     operation.result()
 
     print('Cluster {} successfully deleted.'.format(cluster_name))
-    # [END dataproc_quickstart]
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(
```

Comment:
This isn't tested at all - if we are going to add CLI functionality, we should have some tests verifying that it still works. 

Alternatively, I'd rather avoid CLI and instead give the user somewhere at the top of the code to set the variables instead.

Reply:
I can add a test for this, but the motivation for CLI is to make this a runnable tool as-is without needing to modify code.